docs/GettingStarted.md

```diff
@@ -719,6 +719,32 @@ overhead.
     sampler = Datadog::RateSampler.new(0.5) # sample 50% of the traces
     Datadog.tracer.configure(sampler: sampler)
 
+#### Priority sampling
+
+Priority sampling consists in deciding if a trace will be kept by using a priority attribute that will be propagated for distributed traces. Its value gives indication to the Agent and to the backend on how important the trace is.
```

Comment:
I think it's important to outline that priority sampling is interesting in the case of distributed tracing. Something as `Priority sampling should be turned on when using [distributed tracing](#Distributed_Tracing), as it ensures all parts of a given trace are kept together and sent to the backend.` maybe ? Just a thought.

Reply:
@ufoot I think that's a good idea. Do you think we should add that paragraph after the current one or replaced it altogether?